podpac/core/coordinates/test/test_coordinates1d.py

```diff
@@ -108,15 +111,22 @@ def test_horizontal_resolution(self):
         assert lat.horizontal_resolution(lat, ell_tuple, coord_name) == 4442569.935968436 * podpac.units("meter")
         assert lon.horizontal_resolution(lat, ell_tuple, coord_name) == 0.0 * podpac.units("meter")
 
-        # Resolution: summary
-        assert lat.horizontal_resolution(lat, ell_tuple, coord_name, restype="summary") == (
-            4442569.935968436 * podpac.units("meter"),
-            13040.905617921147 * podpac.units("meter"),
-        )
-        assert lon.horizontal_resolution(lat, ell_tuple, coord_name, restype="summary") == (
-            5558704.3695234 * podpac.units("meter"),
-            3399219.0171971265 * podpac.units("meter"),
-        )
+        # # Resolution: summary
+        lat_summary = lat.horizontal_resolution(lat, ell_tuple, coord_name, restype="summary")
+        lon_summary = lon.horizontal_resolution(lat, ell_tuple, coord_name, restype="summary")
+        lat_compare = (
+                4442569.935968436 * podpac.units("meter"),
+                13040.905617921147 * podpac.units("meter"),
+            )
+        lon_compare = (
+                5558704.3695234 * podpac.units("meter"),
+                3399219.0171971265 * podpac.units("meter"),
+            )
+        for i in range(2):
+            assert lat_summary[i].units==lat_compare[i].units
```

Comment:
@mpu-creare the approximate compare couldn't be applied to the `pint.utils.Quantity` directly, this was my attempt at achieving the same result while also respecting units (the units of assignment, not converting to a common unit).

Reply:
Great fix.